src/Contact/View/page.twig

```diff
@@ -1,5 +1,28 @@
 {% include 'Common/View/header.twig' %}
-<section class="container">
-        <h1>Kontakt</h1>
+<section class="container contact-section">
+<form method="post" action="" class="form">
+<div>
+<h1 input class="form-title">NAPISZ DO NAS</h1>
+
+        <div>
```

Comment:
Remove the div wrapper for all inputs if you do not need this (for example to add styles)